### ISSUE TITLE
Add Tax Map Basemap option

### ISSUE THREAD

**Issue by croswell81** [NYCPlanning/labs-zola#451]
Add the Tax Map Basemap into Zola. Users want to be able to see tax lot info (dimensions, etc.) alongside the zoning layers.

In the “old” ZoLa, you can click on “Tax Map” in the drop down menu for “Map Type” where you could see lot dimensions.   In the “new” ZoLa, you can only view the Tax Map when you click on the “Digital Tax Map” link that forwards you to DOF.  However, you lose the ability to view all the layers of ZoLa while looking at the tax map.   She suggests adding the Tax Map as an option into the “Basemaps” drop down menu.
-Zoning Help Desk Caller (via Scott Solomon, DCP)

**Comment by chriswhong** [comment]
The core of the issue is that old ZoLa uses a custom tile grid based on NY state plane, so the tiles they have available for the digital tax map are not compatible with MapboxGL (which wants standard webmercator tiles)

We attempted a very involved workaround that involved stitching together, clipping, and shrinking DTM tiles.  We got them to work, but the process was too cpu-intensive and took too long to render tiles.

Closing this as wontfix until there is a compatible raster tileset available.

**Comment by timkeane** [comment]
DoITT already has a project lined up to create a new DTM tile set in web mercator.  I hope to have this available by the end of Q1 2018.

**Comment by michaelparkinson** [comment]
Want to echo the sentiment that it would be helpful to have DOF tax block/lot info as a base map. It is an extremely useful tool that many borough planners rely on for quick research/recon.

**Comment by croswell81** [comment]
Two more public feedback requests related to this issue:

Liz Genovese <lizg@nyappraisal.com> The new maps look great, however there is one change that makes the maps less useful.  The lot dimensions are not longer on the maps when you add the layer for the tax map.  This was a feature available on the old Zola.  If you would add the lot dom4ensions as a feature for the tax lots that would be a great help.

Steven Manela <stevenm@nyappraisal.com>The new maps look great, however there is one change that makes the maps less useful.  The lot dimensions are no longer on the maps when you add the layer for the tax map.  This was a feature available on the old Zola.  
If you would add the lot dimensions as a feature for the tax lots that would be a great help.

**Comment by hannahkates** [comment]
@timkeane Is the new DTM tile set in web mercator now available? If so, can someone point us to the link for it? Thanks!

**Comment by timkeane** [comment]
Wish it were. We are still waiting on a P.O. for software. How long ago did
I say we would have it soon? A year? Or is it more?

**Comment by hannahkates** [comment]
Gotcha, thanks for the update. We'll likely follow up at some point to discuss other options for ZoLa and another project that will use the tax map tiles.

**Comment by allthesignals** [comment]
@timkeane wanted to ask, is the new DTM tile set in web mercator now available? Any updates? Thank you!

**Comment by timkeane** [comment]
I wish I had an update. If I was forced to guess, I would say within 6
months, but I've said stuff like that before. Mostly depends on decisions
out of our control. Fingers crossed.

**Comment by allthesignals** [comment]
No problem, I understand @timkeane! Thanks for keeping us updated :)
